OncocoreVS.fsh

```diff
@@ -62,15 +62,17 @@ Description:  "The grade of the tumor. This is a subset of the LOINC answer list
 * SCT#60815008  "Grade not determined (finding)" // synonyms include not applicable, not stated
 //* SCT#12619005  "GX grade (finding)" // includes synonym "Grade cannot be assessed", this is the terminology used by AJCC
 
-ValueSet:   ConditionStatusTrendVS
-Title: "Condition Status Trend Value Set"
-Description:  "Standardized value set that describes the trend or status of a disease, condition, or ability. Based on LOINC answer lists LL2751-7 and LL4721-8. The value SNOMED CT 260415000 (Not Detected) should be used to express No Evidence of Disease (NED), condition resolved, or full remission."
-* SCT#260415000       "Not detected. Use this term for expressing NED (no evidence of disease), condition resolved, or full remission."   //* SCT#723506003 "Resolved" or * SCT#103338009 "In full remission"
-* SCT#385633008       "Improving"
-* SCT#58158008        "Stable"
-* SCT#230993007       "Worsening"
-* SCT#42425007	    "Equivocal. Equivocal represents a borderline value, too close to call; for example, a value very close to a cut-off between positive and negative."
-* SCT#82334004        "Indeterminate. Indeterminate means the results were uninterpretable, or cannot be determined; technical issues prevented obtaining a valid result."
+ValueSet:   CancerStatusTrendVS
```

Comment:
I'm very concerned about this change. Is there a specific ballot comment that compelled it? If not, recall we had detailed conversations about the values, and they were picked carefully with consultation with ASCO. I'm also afraid of affecting ICAREdata. They have had a hard time mapping their values to ours. There have been recent discussions with EPIC about this -- in fact there is one scheduled for today. We risk setting this back to the start.  ![image](https://user-images.githubusercontent.com/492232/73206885-c85cdd00-4111-11ea-92f9-3e8db5eff750.png)

Reply:
You might be looking at an old picture. The value set changes were what Rute and Carmela wanted based on iCAREdata and discussions with Epic.  cc: @rutemartins to comment.  We'll do further changes based on her input.

Reply:
I don't think its old. Mary sent this around last Thursday. But waiting also on @rutemartins and today's meeting.